tools/devops/device-tests/templates/device-tests.yml

```diff
@@ -279,7 +279,6 @@ steps:
     fi
   env:
     WORKING_DIR: $(System.DefaultWorkingDirectory) 
-    XAMARIN_STORAGE_PATH: $(XAMARIN_STORAGE_PATH)
     TESTS_EXTRA_ARGUMENTS: ${{ parameters.testsLabels }}
     USE_XAMARIN_STORAGE: '${{ parameters.useXamarinStorage }}'
     VSDROPS_URI: '${{ parameters.vsdropsPrefix }}/$(Build.BuildNumber)/$(Build.BuildId);/tests/' # uri used to create the vsdrops index using full uri
```

Comment:
Double quotes to expand "$(Build.BuildId)?

Reply:
This is different, this is not bash, is yaml